src/stores/PreferenceStore/PreferenceStore.ts

```diff
@@ -703,25 +728,24 @@ export class PreferenceStore {
 
         // set preference in variable
         if (key === PreferenceKeys.LOG_EVENT) {
-            if (!Event.isEventTypeValid(value)) {
+            if (!Event.isTypeValid(value)) {
                 return false;
             }
-            let eventList = this.preferences.get(PreferenceKeys.LOG_EVENT);
-            if (!eventList || !Array.isArray(eventList)) {
-                eventList = [];
-            }
-            if (eventList.includes(value)) {
-                eventList = eventList.filter(e => e !== value);
-            } else {
-                eventList.push(value);
-            }
+            let eventList = this.setCheckBoxList(this.preferences.get(PreferenceKeys.LOG_EVENT), value);
```

Comment:
Use `const` if the variable won't be changed.

Reply:
okay.